### DISCUSSION TITLE
Gradio Neutral Theme bricked my Automatic

### DISCUSSION THREAD

**Discussion by lyonstyle001** [vladmandic/sdnext#210] [Q&A] [answered]
is there anyway i can reset this? i cant start to open my webui in browser after changing a theme. i tried to search for similar issues but it only directs me to gradio site and im really clueless on how to edit anything. please help

`Loading theme: jkorstad/Neutral
Traceback (most recent call last)
│ D:\AI\automatic\launch.py:94 in <module>
│
│93     import webui
│ > 94     webui.webui()
│95
│
│D:\AI\automatic\webui.py:201 in webui
│
│200 def webui():
│ > 201     start_ui()
│202     load_model() 
│
│D:\AI\automatic\webui.py:175 in start_ui
│
│174
│ > 175     app, _local_url, _share_url = shared.demo.launch(
│176         share=cmd_opts.share,
│
│ D:\AI\automatic\venv\lib\site-packages\gradio\blocks.py:1422 in launch
│
│1421         if not self.enable_queue and self.progress_tracking:
│ > 1422             raise ValueError("Progress tracking requires queuing to be enabled.")
│1423

ValueError: Progress tracking requires queuing to be enabled.
Press any key to continue . . .`

**Comment by vladmandic** [comment, marked as answer]
> self.enable_queue

it's not gradio theme, its my typo in the latest update, fixed. sorry about that.

**Reply by lyonstyle001** [reply]
thank you for responding and that was really fast, cheers to you @vladmandic
